commit 1afcf4e9427a6ba4300c5d62c4ae9c913f2ecbdb
Author: Sage Weil <sweil@redhat.com>
Date:   Fri Jun 16 17:15:37 2017 -0400

    mgr/DaemonServer: print PGMapDigest summary on send
    
    Useful for debugging.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index a4f53333e93..178a7082c3d 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -850,7 +850,7 @@ void DaemonServer::send_report()
 	  // FIXME: no easy way to get mon features here.  this will do for
 	  // now, though, as long as we don't make a backward-incompat change.
 	  pg_map.encode_digest(osdmap, m->get_data(), CEPH_FEATURES_ALL);
-
+	  dout(10) << pg_map << dendl;
 	  pg_map.get_health(g_ceph_context, osdmap,
 			    m->health_summary,
 			    &m->health_detail);
